air/src/air/context.rs

```diff
@@ -163,32 +166,10 @@ impl<B: StarkField, P> AirContext<B, P> {
             trace_domain_generator: B::get_root_of_unity(trace_length.ilog2()),
             lde_domain_generator: B::get_root_of_unity(lde_domain_size.ilog2()),
             num_transition_exemptions: 1,
-            logup_gkr: false,
+            logup_gkr,
```

Comment:
One more question: do we actually need `logup_gkr` field in `AirContext`. Seems like we should be able to get it from `trace_info` which is available in `AirContext` already.

Reply:
Agreed, removed it now